core/internal/features_test.go

```diff
@@ -930,6 +930,164 @@ func TestIntegration_ExternalInitiator_WithMultiplyAndBridge(t *testing.T) {
 	assert.Equal(t, `{"result": "4200000000"}`, finalResult.Data.Raw)
 }
 
+func TestIntegration_ExternalInitiatorV2(t *testing.T) {
+	t.Parallel()
+
+	ethClient, _, assertMockCalls := cltest.NewEthMocksWithStartupAssertions(t)
+	defer assertMockCalls()
+
+	app, cleanup := cltest.NewApplication(t,
+		ethClient,
+		cltest.UseRealExternalInitiatorManager,
+	)
+	defer cleanup()
+
+	app.Config.Set("TRIGGER_FALLBACK_DB_POLL_INTERVAL", "10ms")
```

Comment:
What is this needed for?

Reply:
job.Spawner picks up jobs asynchronously, and doesn't respond to PG events within transactional tests